input/templates/partials/catalog/reserve/store-3.json

```diff
@@ -6,9 +6,8 @@
             "$ref": "../../common/address-1.json"
         },
         "openingTimes": "<p>Mo-Fr. 10:00 a.m - 8:00 p.m.<br/>Sa. 9:00 a.m - 6:00 p.m</p>",
-        "location": "48.1536723,11.5805786"
+        "location": "48.1536723, 11.5805786"
```

Comment:
The only thing that bothers me is the format of the location, instead of a string which needs to be parsed
this would be much better to work with and pass around:

```json
"location" : {
  "lat" : 48.1536723,
  "lng" : 11.5805786
}
```